Library/Homebrew/keg.rb

```diff
@@ -423,18 +422,17 @@ def resolve_any_conflicts(dst, mode)
       return
     end
 
-    if stat.directory?
-      begin
-        keg = Keg.for(src)
-      rescue NotAKegError
-        puts "Won't resolve conflicts for symlink #{dst} as it doesn't resolve into the Cellar" if ARGV.verbose?
-        return
-      end
-
-      dst.unlink unless mode.dry_run
-      keg.link_dir(src, mode) { :mkpath }
-      return true
+    return unless stat.directory?
+    begin
+      keg = Keg.for(src)
+    rescue NotAKegError
+      puts "Won't resolve conflicts for symlink #{dst} as it doesn't resolve into the Cellar" if ARGV.verbose?
```

Comment:
Better to try and keep this below 80 chars by e.g. splitting this into an`if .. end`.